packages/dts-test/defineComponent.test-d.tsx

```diff
@@ -1364,12 +1364,11 @@ describe('function syntax w/ runtime props', () => {
   )
 
   defineComponent(
-    (_props: { msg: string }) => {
+    (_props: { msg: number }) => {
       return () => {}
     },
     {
       props: {
-        // @ts-expect-error prop type mismatch
```

Comment:
Actually, it's by design. We expected there was an error, and this directive is to ensure the error is emitted.

Reply:
Sorry, rolled back here